cli/dcoscli/data/marathon-group-schema.json

```diff
@@ -143,7 +137,8 @@
                 },
                 "dependencies": {
                     "items": {
-                        "$ref": "#/definitions/pathType"
+                        "pattern": "^(\\/?((\\.{2})|([a-z0-9\\-]*))($|\\/))*$",
+                        "type": "string"
```

Comment:
I just moved the pathType info here, since it's only used once, and this makes things simpler.